libs/CCAppCommon/src/ccApplicationBase.cpp

```diff
@@ -76,8 +76,8 @@ void ccApplicationBase::initOpenGL()
 		QSurfaceFormat::setDefaultFormat(format);
 	}
 
-	// The 'AA_ShareOpenGLContexts' attribute must be defined BEFORE the creation of the Q(Gui)Application
-	// DGM: this is mandatory to enable exclusive full screen for ccGLWidget (at least on Windows)
+	//! The 'AA_ShareOpenGLContexts' attribute must be defined BEFORE the creation of the Q(Gui)Application
+	//! DGM: this is mandatory to enable exclusive full screen for ccGLWidget (at least on Windows)
```

Comment:
Same, we don't use //! in cpp files

Reply:
+ same below

Reply:
ok thanks I'll fix these. I'll take note of the documentation styles.